### PR TITLE
Incall vibration options [2/3]

### PR DESCRIPTION
**Pull request by MASTERGUY**
*allow setting vibration when call is connected
*allow setting vibration when call is disconnected
*allow setting vibration for call waiting
*this works with google and aosp dialer :)

### CODE REVIEW

**Comment by poad42** [comment]
-1 Fix authorship on this.

**Comment by MASTERGUY** [comment]
> -1 Fix authorship on this.

oooof sorry i forgot about it

**Comment by idoybh** [comment]
Who has the original authorship?

**Comment by idoybh** [comment]
Same about tabs...

File: src/com/android/server/telecom/Ringer.java [hunk 1]
```diff
@@ -519,4 +525,10 @@ private boolean getVibrateWhenRinging(Context context) {
             || (mSystemSettingsUtil.applyRampingRinger(context)
                 && mSystemSettingsUtil.enableRampingRingerFromDeviceConfig());
     }
+	public void vibrate(int v1, int p1, int v2) {
```
Comment (idoybh) [inline src/com/android/server/telecom/Ringer.java]: space

**Comment by NurKeinNeid** [comment]
Thank you, guys! Fixed and merged. :wink:

**Pull request closed without merging**